src/commands/classes/_common.ts

```diff
@@ -0,0 +1,89 @@
+import ExecutionContext from '../../ExecutionContext';
+import * as Discord from '../../Discord';
+
+export type dbClass = {
+  name: string;
+  description: string;
+  id: string;
+  members: string[];
+};
+
+export const classExists = async (
+  ctx: ExecutionContext,
+  guildId: string,
+  name: string,
+): Promise<boolean> => {
+  const db = await ctx.db.getDb(guildId);
+
+  const classObj = await db.collection('classes').findOne({ name: name });
```

Comment:
Instead of hard-coding the string database name, can we extract it to a constant in this file?
https://github.com/nu25/hoosky-bot/blob/558d1510b97050a8be926a14d36b3f874f507b52/src/database/constants.ts#L3-L8
And replace it like so:
```suggestion
  const classObj = await db.collection(Collection.CLASSES).findOne({ name: name });
```